app/javascript/styles/mastodon/components.scss

```diff
@@ -11044,7 +11044,7 @@ noscript {
   overflow: hidden;
   flex-shrink: 0;
   border-bottom: 1px solid var(--background-border-color);
-  touch-action: pan-x;
+  touch-action: pan-y;
```

Comment:
TIL! So basically, this declaration specifies which direction we want to keep being handled natively, while horizontal scrolling is prevented & handled by JS? Seems like it could use a comment as it's very counter-intuitive.